doc/DUMP-itstar.md

```diff
@@ -2,13 +2,12 @@
 
 ### Saving ITS files and extract them using itstar
```

Comment:
Other tweaks to the file, while you're updating it:

- "and extracting" here
- Typo below: "directores"
- Is it worth mentioning doing "rewind" before "load"? (I usually need to do this when I'm reading in multiple files...)